components/nexus-core/src/main/java/org/sonatype/nexus/proxy/storage/remote/httpclient/HttpClientRemoteStorage.java

```diff
@@ -511,16 +511,24 @@ HttpResponse executeRequest(final ProxyRepository repository, final ResourceStor
     if (outboundRequestLog.isDebugEnabled()) {
       stopwatch = new Stopwatch().start();
     }
+
+    HttpResponse response = null;
     try {
-      return doExecuteRequest(repository, request, httpRequest, contentRequest);
+      response = doExecuteRequest(repository, request, httpRequest, contentRequest);
     }
     finally {
       timerContext.stop();
       if (stopwatch != null) {
-        outboundRequestLog
-            .debug("[{}] {} {} - {}", repository.getId(), httpRequest.getMethod(), httpRequest.getURI(), stopwatch);
+        outboundRequestLog.debug("[{}] {} {} -> {}; {}",
```

Comment:
do we need the semi colon and arrow?

not against this new format, but just curious mostly why you added that and the arrow.